src/sysinfo.cc

```diff
@@ -12,58 +12,53 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "internal_macros.h"
-
 #ifdef BENCHMARK_OS_WINDOWS
```

Comment:
Where is the definition of `BENCHMARK_OS_WINDOWS` coming from if not `internal_macros.h`?